module/zstd/zstd.c

```diff
@@ -629,7 +629,7 @@ zstd_mempool_deinit(void)
 	    ZSTD_POOL_MAX * sizeof (struct zstd_pool));
 }
 
-int __init
+extern int __init
```

Comment:
Didn't I warn you about this? ;)

Reply:
did you?

Reply:
I wonder if the others have to be extern, too....

Reply:
thats not the bug. doesnt matter if you write extern or not.

Reply:
Yeah, I wrote "dont forget this" on the zstd_init (but basically also about adding the extern back)